tests/integration.rs

```diff
@@ -3295,10 +3355,8 @@ mod tests {
 
         #[test]
         fn test_boot_from_virtio_pmem() {
-            let mut focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+            let mut focal = UbuntuDiskConfig::new(UbuntuImageType::Focal, true);
```

Comment:
I think it would be better to have `UbuntuImageType::FocalPmem` and do away with this boolean.

Reply:
In fact rather than `FocalPmem` i would say `FocalRawFile`

Reply:
Fixed!